changelog/unreleased/kong/bump-atc-router.yml

```diff
@@ -0,0 +1,3 @@
+message: "Bumped atc-router from v1.6.2 to v1.7.0"
```

Comment:
Please update the message to include a summary of what has been changed between these two versions.

Reply:
done.